jujugui/options.py

```diff
@@ -21,20 +21,22 @@ def update(settings):
     """
     _update(settings, 'jujugui.apiAddress', default=None)
     _update(settings, 'jujugui.auth', default=None)
-    _update(settings, 'jujugui.base_url', default='')
-    _update(settings, 'jujugui.bundleservice_url',
-            default=DEFAULT_BUNDLESERVICE_URL)
-    _update(settings, 'jujugui.charmstore_url', default=DEFAULT_CHARMSTORE_URL)
     _update(settings, 'jujugui.controllerSocketTemplate', default='/api')
     _update(settings, 'jujugui.GTM_enabled', default=False)
     _update(settings, 'jujugui.password', default=None)
-    _update(settings, 'jujugui.plans_url', default=DEFAULT_PLANS_URL)
-    _update(settings, 'jujugui.payment_url', default=DEFAULT_PAYMENT_URL)
-    _update(settings, 'jujugui.terms_url', default=DEFAULT_TERMS_URL)
     _update(settings, 'jujugui.socketTemplate', default='/model/$uuid/api')
-    _update(settings, 'jujugui.static_url', default='')
     _update(settings, 'jujugui.stripe_key', default='')
     _update(settings, 'jujugui.user', default=None)
+    _update(settings, 'jujugui.static_url', default='')
```

Comment:
This should use update_url as well, but fixing all the places it's used is out of scope for this fix, which is blocking release. We'll get it as a follow up.